src/SortableContainer/index.js

```diff
@@ -148,6 +151,9 @@ export default function sortableContainer(
         this.scrollContainer = useWindowAsScrollContainer
           ? this.document.scrollingElement || this.document.documentElement
           : this.container;
+        this.keyScrollContainer = useWindowAsScrollContainer
```

Comment:
I was having issues specifically with the autoscroll in the nested list example. Moving an item within one of the categories wouldn't trigger the autoscroll since `this.scrollContainer` pointed to the category, not the larger scrolling list itself. 

This isn't really an issue with the mouse scrolling, but for keyboard scrolling you'd have the helper spilling over the outside of the container. I wasn't sure whether to change the initialization of `this.scrollContainer` and possibly change the existing mouse sorting implementation, or to add a separate pointer for my own purposes, so I opted for what I thought was the safer route until I could get your thoughts on the matter 😅

Reply:
Follow up: made a PR for this subissue #507